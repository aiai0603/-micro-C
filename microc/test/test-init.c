/*
bug:全局变量不能附初值
Int  i =1;
*/
Void main(Int n) {
  
    Int a = 7;
    Char b = 'c';
    String s = "12121212";
    Float f = 12.32;
    
    //print("%d",i);
    print("%d",a);
    print("%c",b);
    println(s);
    print("%f",f);

}

