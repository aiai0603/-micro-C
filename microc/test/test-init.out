24 19 1 5 25 13 0 1 1 0 7 12 13 0 2 1 27 99 12 13 0 3 1 26 1095048888 12 13 0 1 1 11 22 15 -1 13 0 2 1 11 23 15 -1 13 0 3 1 11 31 21 4